Underline Strikethrough.roboFontExt/lib/Underline StrikeThrough.py

```diff
@@ -539,7 +551,12 @@ def setAllButtonCallback(self, sender):
         for font in self.fonts:
             fontIdentifier = self.getFontIdentifier(font)
             font.info.postscriptUnderlineThickness = self.roundInteger(uT[fontIdentifier])
-            font.info.postscriptUnderlinePosition  = self.roundInteger(uP[fontIdentifier])
+            # Math only works if there is a underline thickness
+            if uT[fontIdentifier]:
+                font.info.postscriptUnderlinePosition  = self.roundInteger(uP[fontIdentifier] - uT[fontIdentifier] / 2)
+            else:
+                font.info.postscriptUnderlinePosition = None
```

Comment:
Not sure, but maybe would rather have some value? With no thickness, it's theoretically the same right?
```python
else:
    font.info.postscriptUnderlinePosition = self.roundInteger(uP[fontIdentifier])
```